pkg/agent/config/node_config.go

```diff
@@ -33,6 +33,17 @@ const (
 	BridgeOFPort = 0xfffffffe
 )
 
+const (
```

Comment:
Do we have a better location for these? node_config.go is mostly for configs now.

Reply:
Moved to pkg/agent/types/marks.go, following https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/types/labels.go